views/partials/_header.njk

```diff
@@ -53,10 +53,8 @@
       <label class="govuk-visually-hidden" for="app-site-search__input">Search Design system</label>
       <a class="app-site-search__link govuk-link" href="/sitemap/">Sitemap</a>
     </div>
-    <div class="app-header-mobile-nav-toggler-wrapper">
-      <button id="app-mobile-nav-toggler" class="govuk-button app-header-mobile-nav-toggler js-app-mobile-nav-toggler" aria-controls="app-mobile-nav">
-        Menu
-      </button>
-    </div>
+    <button hidden="true" class="govuk-header__menu-button js-app-navigation__toggler" aria-controls="app-navigation">
```

Comment:
As [`hidden`](https://html.spec.whatwg.org/multipage/interaction.html#the-hidden-attribute) is a [boolean attribute](https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#boolean-attribute) in HTML, it should not have a value.

```suggestion
    <button hidden class="govuk-header__menu-button js-app-navigation__toggler" aria-controls="app-navigation">
```